packages/dashboard/src/components/DetailPanel/BaseContainer.tsx

```diff
@@ -0,0 +1,130 @@
+import * as React from 'react';
+import { Panel, PanelType } from 'office-ui-fabric-react/lib/Panel';
+import { IDetailPanelBaseCommonAction, IBaseContainerProps } from './DetailPanel.types';
+import { MessageBanner } from './Body/MessageBanner';
+import { ActionBar } from './Footer/ActionBar';
+import { IconButton } from 'office-ui-fabric-react/lib/Button';
+import { Header } from './Header/index';
+import { detailPanelBaseStyles } from './DetailPanel.styles';
+import { AnimationClassNames, mergeStyles } from 'office-ui-fabric-react/lib/Styling';
+
+type BodyContainerType = IBaseContainerProps & IDetailPanelBaseCommonAction;
+
+const baseContainer: React.SFC<BodyContainerType> = (props: BodyContainerType) => {
```

Comment:
I thought `React.SFC` was deprecated?

https://github.com/DefinitelyTyped/DefinitelyTyped/pull/30364

Reply:
The type def of dashboard project was not updated yet...